sr_port/mumps.hlp

```diff
@@ -217,11 +222,11 @@
      * Establish "watch points" with triggers to trap incorrect accesses on
        global variable updates.
 
-   The following table summarizes the GT.M language extensions that
+   The following table summarizes the YottaDB language extensions that
    facilitate debugging.
 
    +------------------------------------------------------------------------+
-   |                          GT.M Debugging Tools                          |
+   |                          YottaDB Debugging Tools                          |
```

Comment:
There are a few places like this one where the last column in the table is not aligned with surrounding lines due to the GT.M -> YottaDB change. Can those be fixed?

Reply:
I found/fixed 6 places that I had missed (out of many dozens).